Sources/Camille/AnnouncementBot.swift

```diff
@@ -0,0 +1,43 @@
+import Bot
+import Sugar
+
+enum AnnouncementMessage {
```

Comment:
Whats the function of the enum? - I think maybe we could make a Configuration struct (see KaramaBot) that takes a string or ChatPostMessage as an input